Eshop.Domain/Product.cs

```diff
@@ -1,24 +1,28 @@
-﻿namespace Eshop.Domain
+﻿using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Eshop.Domain
 {
     public class Product
     {
         public Product(int id, string title, string description, decimal price, Category category)
         {
             if (id < 0)
             {
-                throw new ArgumentOutOfRangeException("id");
+                throw new ArgumentOutOfRangeException(nameof(id));
```

Comment:
Good improvement, how did you come with it?

Reply:
AI